CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - [IMPROVEMENT] Reduce number of view updates by filtering events from payload. See [#1328][]
+- [BUGFIX / BREAKING] Do not propagate attributes from Errors and LongTasks to Views.
```

Comment:
Thanks for updating the CHANGELOG 👌, however this diff doesn't look healthy. Given that [CHANGELOG on develop](https://github.com/DataDog/dd-sdk-ios/blob/develop/CHANGELOG.md) now includes `2.0.0`, this should go alone into "Unreleased" category on top of 2.0.0. This diff shows it differently. Please rebase 🙏.